source/basic.tex

```diff
@@ -4202,6 +4203,9 @@
 \item
 when throwing an exception\iref{except.throw}.
 \begin{note}
+A temporary object is unnamed.
+\end{note}
```

Comment:
This is the wrong place for such a note.

The note that follows is talking about the exception object, which needs to attach to the last bullet.

Whereas the new note applies to the entire bulleted list and thus need to be after the bulleted list.

Reply:
I've moved the note down. Now it's immediately above the original [Note 2](https://eel.is/c++draft/class.temporary#note-2).